photon-lib/src/main/scala/com/linkedin/photon/ml/hyperparameter/estimators/kernels/StationaryKernel.scala

```diff
@@ -56,6 +58,22 @@ abstract class StationaryKernel(
    */
   protected[kernels] def fromPairwiseDistances(dists: DenseMatrix[Double]): DenseMatrix[Double]
 
+  /**
+   * Unwraps hyper-parameters from scaled values to true values
+   *
+   * @param x the matrix of points, where each of the m rows is a point in the space and some columns contain scaled values
+   * @return the unwrapped x that all the columns have true values
+   */
+  protected[kernels] def unwrap(x: DenseMatrix[Double]): DenseMatrix[Double] = {
```

Comment:
I think "transform" would be a better name for this function, because from the point of view of the kernel, the values are being transformed. It's not really aware of any prior "wrapping".